src/apps/weblib/interface.js

```diff
@@ -1,169 +1,44 @@
+function Method(name, ...args) {
+  const argList = [...args].join(', ')
+  // eslint-disable-next-line no-new-func
```

Comment:
We can use Function() here, because this JS will only be executed in compile time. Emscripten only calls toString() on each property in the object in mergeInto() to generate cvizzu.js. The generated code looks like this:
```javascript
  function _canvas_addPoint(canvas,x,y) {
  Module.renders[canvas].addPoint(x, y)
  }

```